test/unit/test_amqpclient.js

```diff
@@ -732,15 +257,15 @@ describe('AMQPClient', function() {
     });
 
     it('should only create a single connection, session, multiple links for multiple receives', function() {
```

Comment:
This test also doesn't really make any sense anymore since creating a link no longer spawns the connection/session